bindings/python/tests/test_pt_comparison.py

```diff
@@ -119,6 +127,18 @@ def test_gpu(self):
         reloaded = load_file(local)
         self.assertTrue(torch.equal(torch.arange(4).view((2, 2)), reloaded["test"]))
 
+    @unittest.skipIf(not npu_present, "Npu is not available")
+    def test_npu(self):
+        data = {
+            "test": torch.arange(4).view((2, 2)).to("npu:0"),
```

Comment:
Can we add a few different dtypes ?
at least f16/bf16/f32 ?.
It's not necessarily done everywhere, but it's good to put in test some slightly more exotic dtypes, or at least make sure they fail if the dtype doesn't exist on a particular device.